src/Softeq.NetKit.Chat.Data.Persistent/Repositories/IClientRepository.cs

```diff
@@ -14,6 +14,8 @@ public interface IClientRepository
         Task AddClientAsync(Client client);
         Task UpdateClientAsync(Client client);
         Task DeleteClientAsync(Guid clientId);
+        Task DeleteMemberClientsWithThresholdAsync(Guid memberId, int inactiveHoursThreshold);
+        Task<bool> MemberHasClientsAsync(Guid memberId);
```

Comment:
IsMemberHasClientsAsync maybe better

Reply:
If you think that method should be renamed then better to have DoesMemberHaveClientsAsync

Reply:
renamed